apps/web/src/services/asset-inventory/constants/cloud-service-constant.ts

```diff
@@ -6,6 +6,7 @@ export const CLOUD_SERVICE_PAGE_URL_QUERY_KEY = [
     'service_account',
     'period',
     'filters',
+    'default_filters',
```

Comment:
Question: I think `default_filters` is working in Cloud Service Detail Page only.. Is this constant `CLOUD_SERVICE_PAGE_URL_QUERY_KEY` used for public in both page? (main / detail page)